### PR TITLE
chore: release-note 2021.1-0702

### PR DESCRIPTION
**Pull request by baptistemesta**

### CODE REVIEW

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -235,6 +235,35 @@ Use BPMN 2 importer for model exchange with third-party editors.
 
 == Bug fixes
 
+=== Fixes in Bonita 2021.1-0702 (2021-07-02)
+
+==== Fixes in Bonita Development Suite (Studio and UI Designer)
+
+* UID-453	- ResponsivenessHelp in editor help is not displayed
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]: "ResponsiveHelp" or "Responsiveness Help"?

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -235,6 +235,35 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* UID-369	- Property type changes of target widgets are not mapped when using Switch feature
+* UID-165	- When modifying & save an asset, if you press SUPR, the selected widget is removed
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* UID-165	- When modifying and saving an asset, pressing SUPPR removes the selected widget
```

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -235,6 +235,35 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+
+
+==== Fixes in Bonita Runtime (including Portal)
+
+* RUNTIME-326	- Docker logs should not contain sensitive informations
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]: For all customers using previous versions of Bonita, we clearly expose that their logs contain sensitive informations. Is that critical? Should we check with Saghar?

File: modules/ROOT/pages/release-notes.adoc [hunk 4]
```diff
@@ -235,6 +235,35 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+
+* Spring framework: `5.2.9.RELEASE` -> `5.2.15.RELEASE`
+* Spring boot: `2.3.4.RELEASE` -> `2.3.12.RELEASE`
+* Jackson: `2.11.2` -> `2.12.3`
+* Javassist: `3.18.1-GA` -> `3.27.0-GA`
+* XStream: `1.4.15` -> `1.4.17`
+* Ehcache: `2.10.6` -> `2.10.9.2`
+* javax.servlet-api: `3.0.1` -> `3.1.0`
+* org.apache.taglibs: `1.1.2` -> `1.2.5`
+* Removed: Xmlbeans
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]: New chapter "==== Library removal" for this removed, and not updated, library

File: modules/ROOT/pages/release-notes.adoc [hunk 5]
```diff
@@ -235,6 +235,35 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* UID-453	- ResponsivenessHelp in editor help is not displayed
+* UID-369	- Property type changes of target widgets are not mapped when using Switch feature
+* UID-165	- When modifying & save an asset, if you press SUPR, the selected widget is removed
+
+
+==== Fixes in Bonita Runtime (including Portal)
+
+* RUNTIME-326	- Docker logs should not contain sensitive informations
+
+==== Library upgrade
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
==== Library updates
```

**Review by NathalieC — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/release-notes.adoc (hunk 1 above)
Comment (baptistemesta) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* UID-453	- Responsiveness: Help in editor help is not displayed
```

File: modules/ROOT/pages/release-notes.adoc (hunk 4 above)
Comment (baptistemesta) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion

==== Library removal

* Xmlbeans
```

**Review by NathalieC — APPROVED** [review]

File: modules/ROOT/pages/release-notes.adoc (hunk 3 above)
Comment (pabloalonso) [reply modules/ROOT/pages/release-notes.adoc]:
```suggestion
* RUNTIME-326	- Reduce information in Docker start logs 
```

**Review by pabloalonso — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/release-notes.adoc [hunk 6]
```diff
@@ -235,6 +235,38 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+==== Fixes in Bonita Development Suite (Studio and UI Designer)
+
+* UID-453	- Responsiveness: Help in editor help is not displayed
+* UID-369	- Property type changes of target widgets are not mapped when using Switch feature
+* UID-165	- When modifying and saving an asset, pressing SUPPR removes the selected widget
+
+
+==== Fixes in Bonita Runtime (including Portal)
+
+* RUNTIME-326	- Reduce information in Docker start logs
```
Comment (pabloalonso) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* RUNTIME-326 - Reduce unneeded information in Docker start logs 
```
As proposed by nath :)

**Review by pabloalonso — CHANGES_REQUESTED** [review]

**Comment by pabloalonso** [comment]
It can be merged once modified

**Pull request merged**